src/Serilog.Sinks.MongoDB/LoggerConfigurationMongoDBExtensions.cs

```diff
@@ -122,11 +121,15 @@ public static class LoggerConfigurationMongoDBExtensions
             if (loggerConfiguration == null) throw new ArgumentNullException("loggerConfiguration");
             if (databaseUrl == null) throw new ArgumentNullException("databaseUrl");
 
-            var optionsBuilder = CollectionOptions.SetCapped(true).SetMaxSize(cappedMaxSizeMb * 1024 * 1024);
+            var options = new CreateCollectionOptions
+            {
+                Capped = true,
+                MaxSize = cappedMaxSizeMb * 1024 * 1024
+            };
 
```

Comment:
Yes, a few classes in MongoDB driver API have been refactored since beta1. I didn't make any functional changes, just updated the code to match MongoDB-beta4 Nuget package.